README.md

```diff
@@ -33,6 +33,29 @@ chimp --watch
 ```
 You can also easily change the browser Chimp with `--browser`, e.g. `--browser=phantomjs`
 
+### Installation as gulp/grunt module
+
+```sh
+npm install chimp
+```
+
+### Usage in a gulp task
+
+```sh
+let Chimp = require('chimp');
+let options = require('./my/config/for/chimp');
+options['_'] = [
+	'/my/path/to/node',
+	'/my/path/to/my/project/node_modules/chimp/bin/chimp.js'
+];
+let chimp = new Chimp(options);
+chimp.run(function (err, res) {
```

Comment:
chimp.init will not reply with the result, also it will call exit. The idea is to be able to do stuff with the json from the output after chimp execution on the gulp task.